zeq/event.cpp

```diff
@@ -49,9 +57,38 @@ flatbuffers::Parser& Event::getParser()
     return _impl->parser;
 }
 
-void Event::setData( const ConstByteArray& data, const size_t size )
+void Event::_update()
+{
+    _impl->func( *this );
+}
+
+bool Event::_fromBinary( const void *data, const size_t size )
+{
+    _impl->setData( ConstByteArray((uint8_t *)data, []( const void* ){}), size );
+    return true;
+}
+
+servus::Serializable::Data Event::_toBinary() const
+{
+    servus::Serializable::Data data;
+    data.ptr.reset( _impl->getData(), []( const void* ){});
```

Comment:
The servus::Data class uses a shared_ptr which is not really used correctly ( all uses are deleting the deleter which is at the end a raw ptr - check ZerBuf class _toBinary function as well ).